airflow/bin/cli.py

```diff
@@ -2525,6 +2527,16 @@ class CLIFactory(object):
             ("-cn", "--celery_hostname"),
             help=("Set the hostname of celery worker "
                   "if you have multiple workers on a single machine.")),
+        'without_mingle': Arg(
+            ("--without-mingle"),
+            default=False,
+            help="Don’t synchronize with other workers at start-up",
+            action="store_true"),
+        'without_gossip': Arg(
+            ("--without-gossip"),
+            default=False,
```

Comment:
perhaps it's best to keep these as default=False and add --without-mingle and --without-gossip to our worker commands so that we're compatible next time we need to update airflow

Reply:
lol, I missed this and came to the same conclusion as you did :D